cirq-aqt/cirq_aqt/aqt_sampler.py

```diff
@@ -26,9 +26,10 @@
 import time
 import uuid
 from typing import cast, Dict, List, Sequence, Tuple, Union
+from urllib.parse import urljoin
 
 import numpy as np
-from requests import put
+from requests import put, post, get
```

Comment:
```suggestion
from requests import post, get
```

`put` is not used anymore

Reply:
Thanks! Just caught this too, fix is in :-)